src/sagemaker/tuner.py

```diff
@@ -525,3 +721,49 @@ def stop(self):
 
     def wait(self):
         self.sagemaker_session.wait_for_tuning_job(self.name)
+
+
+def create_identical_dataset_and_algorithm_tuner(parent, additional_parents=None, sagemaker_session=None):
+    """Creates a new tuner by copying the request fields from the provided parent to the new instance of
+        HyperarameterTuner. Followed by addition of warm start configuration with the type as
+        "IdenticalDataAndAlgorithm" and parents as the union of provided list of additional_parents and the parent.
+
+    Args:
+        parent (str): Primary parent tuning job's name from which the Tuner and Estimator configuration has to be copied
+        additional_parents (set{str}): Set of additional parent tuning job's names along with the primary parent tuning
+            job name to be used in warm starting the transfer learning tuner.
+        sagemaker_session (sagemaker.session.Session): Session object which manages interactions with
+                Amazon SageMaker APIs and any other AWS services needed. If not specified, one is created
+                using the default AWS configuration chain.
+
+    Returns:
+        sagemaker.tuner.HyperparameterTuner: New instance of warm started HyperparameterTuner
+    """
+
+    parent_tuner = HyperparameterTuner.attach(tuning_job_name=parent, sagemaker_session=sagemaker_session)
+    return parent_tuner.identical_dataset_and_algorithm_tuner(additional_parents=additional_parents)
+
+
+def create_transfer_learning_tuner(parent, additional_parents=None, estimator=None, sagemaker_session=None):
+    """Creates a new tuner by copying the request fields from the provided parent to the new instance of
+        HyperarameterTuner.
+        Followed by addition of warm start configuration with the type as "TransferLearning" and parents as the
+        union of provided list of additional_parents and the parent.
```

Comment:
ll. 750-751 isn't a full sentence and I'm a little confused as to how it relates to the previous sentence

Reply:
check now